src/form.js

```diff
@@ -701,7 +736,10 @@ define([
          * maintains a hash table to quickly get a mug by its path.
          */
         getAbsolutePath: function (mug, excludeRoot) {
-            if (!mug.options.isControlOnly) {
+            if (mug && !mug.options.isControlOnly) {
```

Comment:
I realize this is a convenience, but it should not be necessary. It's a bug to call this function with anything but a valid mug. It is the caller's responsibility to know that it's passing a valid mug.